library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+            *** Beta-activity <nowiki> {suggestedTag=Finding-frequency, suggestedTag=Finding-amplitude} [Rhythmic activity in the beta frequency range (14-40 Hz). (Source: Beniczky ea 2017, Table 5; Beniczky ea 2013, Appendix S2, S6.)]</nowiki>
                 **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Gamma-activity-morphology {suggestedTag=Finding-frequency,suggestedTag=Finding-amplitude}
+            *** Gamma-activity <nowiki> {suggestedTag=Finding-frequency,suggestedTag=Finding-amplitude} [Rhythmic activity in the gamma frequency range. (Source: Beniczky ea 2017, Table 5.)]</nowiki>
                 **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
-        ** Spike-morphology <nowiki>[A transient, clearly distinguished from background activity, with pointed peak at a conventional paper speed or time scale and duration from 20 to under 70 ms, i.e. 1/50-1/15 s approximately. Main component is generally negative relative to other areas. Amplitude varies.]</nowiki>
+        ** Spike-morphology <nowiki>[A transient, clearly distinguished from background activity, with pointed peak at a conventional paper speed or time scale and duration from 20 to under 70 ms, i.e. 1/50-1/15 s approximately. Main component is generally negative relative to other areas. Amplitude varies. (Source: Beniczky ea 2017, Table 5; Beniczky ea 2013, Appendix S4.)]</nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** Spike-and-slow-wave-morphology <nowiki>[A pattern consisting of a spike followed by a slow wave.]</nowiki>
+        ** Spike-and-slow-wave-morphology <nowiki>[A pattern consisting of a spike followed by a slow wave. (Source: Beniczky ea 2017, Table 5; Beniczky ea 2013, Appendix S4.)]</nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** Runs-of-rapid-spikes-morphology <nowiki>[Bursts of spike discharges at a rate from 10 to 25/sec (in most cases somewhat irregular). The bursts last more than 2 seconds (usually 2 to 10 seconds) and it is typically seen in sleep. Synonyms: rhythmic spikes, generalized paroxysmal fast activity, fast paroxysmal rhythms, grand mal discharge, fast beta activity.]</nowiki>
+        ** Runs-of-rapid-spikes-morphology <nowiki>[Bursts of spike discharges at a rate from 10 to 25/sec (in most cases somewhat irregular). The bursts last more than 2 seconds (usually 2 to 10 seconds) and it is typically seen in sleep. Synonyms: rhythmic spikes, generalized paroxysmal fast activity, fast paroxysmal rhythms, grand mal discharge, fast beta activity. (Source: Beniczky ea 2017, Table 5; Beniczky ea 2013, Appendix S4.)]</nowiki>
```

Comment:
"and it is typically" -- "and the runs are typically"

Reply:
fixed